install/common/build-image.py

```diff
@@ -59,8 +83,11 @@ def main():
                 check=True,
             )
 
+        # Build the container image, and tag it with two tags; the one we calculated
+        # above, and the "latest" tag.
```

Comment:
It's so that whoever loads the container image, can just run it with `podman run ... dangerzone.rocks/dangerzone:latest`, without having to look into `image-id.txt` for a tag. It makes things simpler for the container isolation provider, when it wants to run the image, as well as our scan actions in our CI.

Still, it feels like a code smell to me as well, and it's probably better to make things clearer in the code, and a bit more involved in our CI. I'll try to make an improvement here.